src/index.html

```diff
@@ -0,0 +1,44 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My to do list</title>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.3/font/bootstrap-icons.css"
+    />
+    <link rel="stylesheet" href="./styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <div class="header">
+          <h4>My todo</h4>
+        </div>
+        <div class="new-todo">
+          <div id="todoform">
+            <input
+              type="text"
+              name="newtodo"
+              id="newtodo"
+              placeholder="Add your list..."
+              required
+            />
+            <button id="EnterButton" type="submit">
+              <img src="Assets/enter.png" />
+            </button>
```

Comment:
- [ ] You did a great job in your design however  the image for the add icon is broken  please fix that it will make your page looks nice and professional :+1: 
- [ ] Also no need to implement any functionality as I see you did implement the functionality for add items kindly remove it you will work on it in the next milestone with a better understanding :+1: 

![image](https://user-images.githubusercontent.com/95297251/206814845-be05ff77-d390-49a1-bb0a-fe04d88e5d7b.png)

Reply:
### Thanks @MaganaAsiati ! for the kind review 

 Is the any way you can suggest me to make that image visible???? The image is visible when  running the app using live server but when I run npm start is not visible !! Can you please tell me why??